config/core.json

```diff
@@ -11,7 +11,7 @@
 		"remote-inbox-notifications": false,
 		"shipping-label-banner": true,
 		"store-alerts": true,
-		"unminified-js": false,
+		"minified-js": false,
```

Comment:
Is is fair to say we're doing this as a temporary measure? If not, I'd find this confusing and expect that our core production release would have `minified-js: true`. But, maybe I'm thinking about this wrong.

Reply:
This feature flag only supports the lack of both minified and unminified JS files in the core build. I chose to rename it because by outward appearance, there are no `*.min.js` files (even though the contents are actually minified).

If we were able to include unminified JS or soucemaps, etc, without inflating the core build size - we might be able to remove this temporary measure.